packages/realm-app-importer/src/react-native/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+	"compilerOptions": {
+		"target": "es2018",
```

Comment:
We've moved to ES2020 elsewhere.  Shouldn't that be a global setting?

Reply:
Make a ticket to review tsconfigs.